suites/pacific/cephadm/tier-1_5-1_service-apply-spec.yaml

```diff
@@ -47,304 +47,312 @@
 #       (b) Deploy NFS on node4 using 'nfs-rgw-pool' pool.
 #===============================================================================================
 tests:
+  # - test:
```

Comment:
????

Reply:
Cleared
For testing new changes, I had commented.

Reply:
Uncomment these TC